ext/standaloneusers/managed/SavedSearch_Permissions.mgd.php

```diff
@@ -97,7 +97,7 @@
     'rewrite' => ' ',
     'icons' => [
       [
-        'icon' => 'fa-square-check',
+        'icon' => 'fa-circle-check fa-solid',
```

Comment:
@ufundo my use of `circle-check` vs `square-check` was intentional. Maybe it wasn't the best way to do so, but I wanted to visually distinguish explicit permissions from implied permissions. With this change they will all look the same.

Reply:
The implied ones should have hollow circles, and the granted ones will have solid circles (see screenshot)

Reply:
(Also happy to unbundle the icon change to get the contrast fix merged if this is contentious)

Reply:
^ have unbundled